rtmp-stream.go

```diff
@@ -98,6 +114,67 @@ func (f *RtmpStreamFactory) startStream(conn rtmp.ClientConn, streamName string)
 	}
 }
 
+func (f *RtmpStreamFactory) ParseURLArgument(urlArg string) (string, []*url.URL, error) {
+	var host string
+	var urls []*url.URL
+	var r = regexp.MustCompile(`{{(?P<min>[1-9][0-9]*) (?P<max>[1-9][0-9]*)}}`) // {{123 456}}
+
+	var regexString = "'{{(?P<min>[1-9][0-9]*) (?P<max>[1-9][0-9]*)}}'"
```

Comment:
Duplicate string constant: use the regexString variable as input to regexp.MustCompile()